Packs/CommonWidgets/Scripts/GetLargestInvestigations/GetLargestInvestigations.yml

```diff
@@ -0,0 +1,27 @@
+args:
+- default: false
+  description: The date whose month is the last month we want to fetch the incidents.
+  isArray: false
+  name: to
+  required: false
+  secret: false
+- default: false
+  description: The date whose month is the first month we want to fetch the incidents.
+  isArray: false
+  name: from
```

Comment:
shouldn't this be fromDate?
Same goes for the other yml

Reply:
no, checked this it is definitely `from` and `to`